catalyst/main.py

```diff
@@ -430,13 +436,16 @@ def _main(parser, opts):
 		# catalyst cannot be run as a normal user due to chroots, mounts, etc
 		log.critical('This script requires root privileges to operate')
 
+       if os.uname().sysname == 'Linux':
```

Comment:
Can you please specifically comment here that FreeBSD doesn't have namespaces at this time. If it got namespaces in future, you might want to improve this code.